client/src/ui/PageFooter.tsx

```diff
@@ -0,0 +1,172 @@
+"use client";
+import Link from "next/link";
+import Image from "next/image";
+import { Typography, Divider } from "@mui/material";
+
+/*
+This Page Footer will be responisble for rendering the footer of all pages in the project
+Except for the Login Page and Sign Up Page
+*/
+
+export const PageFooter = () => {
+  return (
+    <footer
+      style={{
+        backgroundColor: "#f9f9fc",
```

Comment:
I don't think there's a background color for the footer component based on the figma file?

Reply:
There was a wrong color I put on, it has been fixed!